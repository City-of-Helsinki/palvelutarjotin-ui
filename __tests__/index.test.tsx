import React from 'react';
import { render } from '@testing-library/react';

import App from '../pages/index';

describe('Index page', () => {
  it('Should match Snapshot', () => {
    const { container } = render(<App />)

    expect(container).toMatchSnapshot()
  })
})