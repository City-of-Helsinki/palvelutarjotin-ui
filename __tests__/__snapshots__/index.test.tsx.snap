// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Should match Snapshot 1`] = `
<div>
  <div>
    <main>
      <h1>
        Welcome to Palvelutarjotin
      </h1>
    </main>
  </div>
</div>
`;
